src/main/scala/io/iohk/ethereum/network/PeerActor.scala

```diff
@@ -284,6 +284,13 @@ class PeerActor(
 
       case GetStatus =>
         sender() ! StatusResponse(Handshaked(initialStatus))
+
+      case PeerActor.StartBlockBroadcast(blockchain) =>
+        val blockBroadcastActor = context.actorOf(
```

Comment:
do we limit somehow number of broadcast actors per connection?
what will happen if actor will die, should we restart him?